WalletWasabi.Gui/Tabs/WalletManager/GenerateWalletViewModel.cs

```diff
@@ -22,7 +22,7 @@ internal class GenerateWalletViewModel : CategoryViewModel
 		public WalletManagerViewModel Owner { get; }
 		public Global Global => Owner.Global;
 
-		public GenerateWalletViewModel(WalletManagerViewModel owner) : base("Generate Wallet")
+		public GenerateWalletViewModel(WalletManagerViewModel owner) : base("Generate New Wallet")
```

Comment:
NACK, it makes it longer, the spelling is correct and the function is not ambiguous.